tests/SampleApps/DotNetCore/NetCore6PreviewMvcApp/appsvc.yaml

```diff
@@ -0,0 +1,3 @@
+version: 1
+
+run: gunicorn myapp.app --workers 5
```

Comment:
Is there a `gunicorn` command for dotnet?

Reply:
we actually do not need this file anymore. I will revert it.